PROCESSORS.md

```diff
@@ -2728,7 +2728,6 @@ In the list below, the names of required properties appear in bold. Any other pr
 | **Heartbeat Interval**               | 30 sec                                                                                          |                  | The processor will ask the sources to send heartbeats with this interval.                                                                                                                                                                                                                                                       |
 | **Max Elements**                     | 20                                                                                              |                  | The maximum number of events a source will batch together and send at once.                                                                                                                                                                                                                                                     |
 | **Max Latency**                      | 10 sec                                                                                          |                  | The maximum time a source will wait to send new events.                                                                                                                                                                                                                                                                         |
-| Connection Retry Interval            | 10 sec                                                                                          |                  | The interval with which a source will try to reconnect to the server.                                                                                                                                                                                                                                                           |
```

Comment:
Why is this removed?

Reply:
Great catch, it seems I was too zealous with my purging. Fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1664/commits/ddacfb2844366f6f53e86664b1ec83cac8e83979